django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -185,6 +187,56 @@
         win.close();
     }
 
+
+    function customEncodeURIComponent(s) {
+        const QUOTE_MAP = {};
+        const specialChars = '":/_#?;@&=+$,[]<>%\n\\';
+
+        for (let i = 0; i < specialChars.length; i++) {
+            const charCode = specialChars.charCodeAt(i);
+
+            // "This checks if the current character code is equal to 10 (which corresponds to the line feed character)";
+
+            if (charCode === 10) {
+                QUOTE_MAP[charCode] = '_0A';
```

Comment:
```suggestion
            // Line feed \n character
            if (charCode === 10) {
                QUOTE_MAP[charCode] = '_0A';
```
I like the comment, think we can condense it a bit

Reply:
> I like the comment, think we can condense it a bit
@sarahboyce thank you for the review, i will make the necessary modifications